.gitlab-ci.yml

```diff
@@ -55,6 +55,32 @@ Push:
     - cd ${HOME}/go/src/github.com/${REPONAME}/${IMAGENAME}
     - sudo make dockerops
 
+baseline-image:
+  
+  when: always
+  stage: baseline
+  tags:
+    - test
```

Comment:
Why we are using `test` tag?

Reply:
We have different tags for different stages like the setup for setting up the cluster, build for building the image test for testing the BDDs, So I consider this satge as `test` stage only so that I don't have to create new tag for this stage and It is somehow close to test task only.

Reply:
okay